### PR TITLE
DRAFT: Small IDA updates based on early C3 feedback

### PR DESCRIPTION
**Pull request by AndrewB9lab**
This PR includes minor content updates to the IDA Week 0 and Week 1 material, and corrects a broken external link.

### Change scope

The changes in this Pull Request include (please tick all that apply):

* [X] Small language/grammar fixes
* [X] Small content fixes 
* [ ] Addition of new content
* [ ] Sample code/command updates
* [ ] Platform fixes
* [ ] Other

### CODE REVIEW

File: academy/1-what-is-cosmos/1-blockchain-and-cosmos.md [hunk 1]
```diff
@@ -149,7 +149,7 @@ The founding **vision** of Cosmos is that of an easy development environment for
...
 
-To achieve this vision and type of network, the ecosystem relies on an **open-source toolkit**, including the [Inter-Blockchain Communication Protocol (IBC)](https://ibcprotocol.org/), its implementation in the [Cosmos SDK](https://v1.cosmos.network/sdk), and [Tendermint](https://tendermint.com/) as the base layer providing distributed state finality. A set of modular, adaptable, and interchangeable tools helps not only to quickly create a blockchain but also facilitates the customization of secure and scalable chains.
+To achieve this vision and type of network, the ecosystem relies on an **open-source toolkit**, including the [Inter-Blockchain Communication Protocol (IBC)](http://ibcprotocol.dev/), its implementation in the [Cosmos SDK](https://v1.cosmos.network/sdk), and [Tendermint](https://tendermint.com/) as the base layer providing distributed state finality. A set of modular, adaptable, and interchangeable tools helps not only to quickly create a blockchain but also facilitates the customization of secure and scalable chains.
```
Comment (xavierlepretre) [inline academy/1-what-is-cosmos/1-blockchain-and-cosmos.md:152]:
```suggestion
To achieve this vision and type of network, the ecosystem relies on an **open-source toolkit**, including the [Inter-Blockchain Communication Protocol (IBC)](https://ibcprotocol.dev), its implementation in the [Cosmos SDK](https://v1.cosmos.network/sdk), and [Tendermint](https://tendermint.com/) as the base layer providing distributed state finality. A set of modular, adaptable, and interchangeable tools helps not only to quickly create a blockchain but also facilitates the customization of secure and scalable chains.
```

**Review by xavierlepretre — CHANGES_REQUESTED** [review]
Minor nit.

**Review by CitMC — APPROVED** [review]

**Review by CitMC — APPROVED** [review]

**Review by CitMC — APPROVED** [review]

File: hands-on-exercise/1-ignite-cli/2-exercise-intro.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ tags:
 
 # Exercise - Make a Checkers Blockchain
 
-The next sections are all about creating a decentralized game of checkers using Ignite CLI and the Cosmos SDK. Each section progressively advances the project. To learn the most efficiently, you should treat each section as _a solution_ to the questions raised at the start of the section.
+The next sections are all about creating a decentralized game of checkers (see rules [here](https://www.ducksters.com/games/checkers_rules.php)) using Ignite CLI and the Cosmos SDK. Each section progressively advances the project. To learn the most efficiently, you should treat each section as _a solution_ to the questions raised at the start of the section.
```
Comment (xavierlepretre) [inline hands-on-exercise/1-ignite-cli/2-exercise-intro.md:12]: @AndrewB9lab Idea from a student.

**Review by CitMC — APPROVED** [review]

**Comment by coldice** [comment]
@xavierlepretre @CitMC is this ready for merge?

**Comment by xavierlepretre** [comment]
> @xavierlepretre @CitMC is this ready for merge?

For me, yes.

**Review by coldice — APPROVED** [review]

**Pull request merged**